xl2times/utils.py

```diff
@@ -53,13 +53,38 @@ def apply_composite_tag(table: datatypes.EmbeddedXlTable) -> datatypes.EmbeddedX
         and table tag simplified.
     """
     if table.defaults:
-        varname = table.defaults
+        defaults = table.defaults
         df = table.dataframe
-        if "attribute" in df.columns:
-            df["attribute"] = df["attribute"].fillna(varname)
+        # Check for ANSWER-style defaults
+        if "=" in defaults:
+            # Split multiple comma-separated defaults
+            if "," in defaults:
+                defaults = defaults.split(",")
+            else:
+                # Make it a list
+                defaults = list([defaults])
+            # Check whether there are invalid values on the list
+            invalid_defaults = [default for default in defaults if "=" not in default]
+            if invalid_defaults:
+                logger.warning(
+                    f"Expected ANSWER-style defaults, got {invalid_defaults}"
+                )
+            defaults = [default.split("=") for default in defaults]
```

Comment:
```suggestion
            defaults = [default.split("=") for default in defaults if '=' in defaults]
```
Perhaps we want to do this for the non invalid defaults?